tasks/urls.py

```diff
@@ -111,6 +111,16 @@
         views.TaskWorkflowConfirmDeleteView.as_view(),
         name="task-workflow-ui-confirm-delete",
     ),
+    path(
+        "<int:task_workflow_pk>/task/create/",
+        views.TaskWorkflowTaskCreateView.as_view(),
+        name="task-workflow-task-ui-create",
+    ),
+    path(
+        "task/confirm-create/<int:pk>",
```

Comment:
For URL pattern consistency with other confirm-create views (for instance, subtask creation uses `"subtasks/<int:pk>/confirm-create/",`):
```suggestion
        "task/<int:pk>/confirm-create/",
```